config/prod/dian-hm-uat-migration-s3.yaml

```diff
@@ -0,0 +1,42 @@
+# Provision a general S3 bucket or a Synapse External Bucket
+# http://docs.synapse.org/articles/custom_storage_location.html
+template_path: "remote/s3-bucket-v2.yaml"
+stack_name: "dian-hm-uat-migration-s3"
+#sceptre_user_data:
+  # (Optional) Synapse IDs for owner.txt file
+  # SynapseIDs: ["1300139", "3390915"]
+parameters:
+  # The Sage deparment (Platform, CompOnc, SysBio, Governance, etc..)
+  Department: "Platform"
+  # The Sage project (Infrastructure, amp-ad, ntap, dream, etc..)
+  Project: "DIAN"
+  # The resource owner
+  OwnerEmail: "it@sagebase.org"
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+
+  # Settings have default values but can be overriden. Set the below
+  # parameters *only* if you want to override the defaults.
+
+  # (Optional) Allow accounts, groups, and users to access bucket. (default is no access).
+  #  GrantAccess:
```

Comment:
you probably want to give someone access

Reply:
yes, done